### PR TITLE
Feature/sd 63148 rr monthly

### PR DESCRIPTION
**Pull request by richard-mladek**

### CODE REVIEW

**Comment by FussyDuck** [comment]
[CLA assistant check](https://bol-claassistant.ecmwf.int:443/ecmwf/eccodes?pullRequest=178) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://bol-claassistant.ecmwf.int:443/ecmwf/eccodes?pullRequest=178) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://bol-claassistant.ecmwf.int:443/check/ecmwf/eccodes?pullRequest=178) it.</sub>

File: definitions/grib2/products_crra.def [hunk 1]
```diff
@@ -75,24 +75,73 @@ concept marsType(unknown) {
...
 concept marsStream(unknown) {
 
     oper = {
-      productDefinitionTemplateNumber = 8;
+      productDefinitionTemplateNumber = 0;
     }
     oper = {
-      productDefinitionTemplateNumber = 0;
+      productDefinitionTemplateNumber = 8;
     }
```
Comment (sebvi) [inline definitions/grib2/products_crra.def:93]: is it just a swap but effectively doing the same thing?

Comment (richard-mladek) [reply definitions/grib2/products_crra.def:93]: yes, not sure how it happened, just sort of beautification to keep the order

File: definitions/grib2/products_crra.def [hunk 2]
```diff
@@ -75,24 +75,73 @@ concept marsType(unknown) {
...
+      outerLoopTypeOfTimeIncrement = 1;
+      outerLoopLengthOfTimeRange = 21;
+    }
+    # accumd
+    dame = {
+      productDefinitionTemplateNumber = 8;
+      outerLoopTypeOfTimeIncrement = 1;
+      outerLoopLengthOfTimeRange = 24;
+    }
 } : no_copy;
```
Comment (sebvi) [inline definitions/grib2/products_crra.def:116]: I would make the marsStream concept read only as we don't want anyone to set it directly

**Comment by shahramn** [comment]
Can you please sign the CLA?

File: definitions/grib2/products_crra.def (hunk 2 above)
Comment (richard-mladek) [reply definitions/grib2/products_crra.def]: done

**Pull request merged**